packages/esm-patient-registration-app/src/patient-registration/section/section-wrapper.component.tsx

```diff
@@ -2,19 +2,29 @@ import React from 'react';
 import styles from '../patient-registration.scss';
 import { Tile } from 'carbon-components-react';
 import { useTranslation } from 'react-i18next';
+import { SectionDefinition } from '../../config-schema';
+import { Section } from './section.component';
 
-export const SectionWrapper = ({ id, children, name, index }) => {
+export interface SectionWrapperProps {
+  sectionDefinition: SectionDefinition;
+  index: number;
+}
+
+export const SectionWrapper = ({ sectionDefinition, index }: SectionWrapperProps) => {
   const { t } = useTranslation();
+
   return (
-    <div id={id}>
+    <div id={sectionDefinition.id}>
       <h3 className={styles.productiveHeading02} style={{ color: '#161616' }}>
-        {index + 1}. {name}
+        {index + 1}. {sectionDefinition.name}
```

Comment:
This seems like a pain for i18n, but I don't really have a better suggestion off-hand. I suppose one "nice" feature of a fixed list of sections is it can just be pushed into the translations.

Reply:
Yeah I'm inclined to do something like `t(sectionDefinitions.name, sectionDefinitions.name)` so that translations can be used for whatever string is provided.